client/src/pages/Restaurant/index.js

```diff
@@ -32,10 +35,17 @@ function Restaurant() {
       <Header />
       {restaurant.name}
       {foods.length}
+      <RedirectButton image={YelpLogo} url={restaurant.yelp_url} />
+      <RedirectButton
+        image={GoogleMapsLogo}
+        url={`https://www.google.com/maps/search/${restaurant.location.coordinates[1]},${restaurant.location.coordinates[0]}`}
+      />
```

Comment:
Coordinates are in the reverse order, so you would end up in some other random place across the globe. So I just reveresed it to be formatted properly in the Google Maps search. Perhaps there's a more elegant solution?

Reply:
There certainly is.
Since `restaurant.location` is an array of two numbers, you can deconstruct it like:
```js
const [x,y] = restaurant.location
```
x will be assigned the restaurant.location[0]
y will be assigned the restaurant.location[1]

See if you can use this idea to make it cleaner :)